source/expressions.tex

```diff
@@ -2610,45 +2610,43 @@
 semantic properties proceed in the following order:
 
 \begin{itemize}
-\item Substitution of template arguments (if any)
-into the \grammarterm{expression} is performed.
-
-\item If the \tcode{noexcept} specifier is present,
-\tcode{E} shall not be a potentially-throwing expression\iref{except.spec}.
-
-\item If the \grammarterm{return-type-requirement} is present, then:
-
-\begin{itemize}
-\item Substitution of template arguments (if any)
-into the \grammarterm{return-type-requirement} is performed.
-
-\item If the \grammarterm{return-type-requirement} is a
-\grammarterm{trailing-return-type},
-%%% FIXME: is -> shall be
-\tcode{E} is implicitly convertible to
-the type named by the \grammarterm{trailing-return-type}.
-If conversion fails, the enclosing \grammarterm{requires-expression}
-is \tcode{false}.
-
-\item If the \grammarterm{return-type-requirement}
-starts with a \grammarterm{constrained-parameter}\iref{temp.param},
-the \grammarterm{expression} is deduced against
-an invented function template \tcode{F}
-using the rules in \ref{temp.deduct.call}.
-\tcode{F} is a \tcode{void} function template
-with a single type template parameter \tcode{T}
-declared with the \grammarterm{constrained-parameter}.
-A \grammarterm{cv-qualifier-seq} \cv{} is formed
-as the union of \tcode{const} and \tcode{volatile} specifiers
-around the \grammarterm{constrained-parameter}.
-\tcode{F} has a single parameter
-whose \grammarterm{type-specifier} is \cv{}~\tcode{T}
-followed by the \grammarterm{abstract-declarator}.
-%%% FIXME: Remove this; if deduction fails, the construct should
-%%% be ill-formed.
-If deduction fails,
-the enclosing \grammarterm{requires-expression} is \tcode{false}.
-\end{itemize}
+\item
```

Comment:
Please can you undo the indentation of these items? It makes it very hard to see what's actually changing here, and generally we don't indent like this.

Reply:
@zygoloid The reformatting bothered me too for the same reason, but @tkoeppe  suggested in past reviews that I fix the formatting to follow the new [guidelines](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines) as I come across it, and it would be odd to fix the modified \items without fixing them all.  What to do?  As for how we generally format, I'm trying to follow those guidelines - did I misunderstand them?

Reply:
Maybe don't introduce too much non-semantic change in important motion applications like this, but keep those changes for pure-editorial changes during regular editorial review work after motions have been merged?

Reply:
@tkoeppe Sounds good.